.github/workflows/Dev_push.yml

```diff
@@ -0,0 +1,53 @@
+name: "Push event"
+on:
+    push:
+        branches:
+            - task_review
+
+jobs:
+    Push-Request:
+        if: github.event_name == 'push'
+        runs-on: ubuntu-latest
+        environment:
+          name: "Dev"
+        steps:
+          - name: Checkout Repo
+            uses: actions/checkout@v4
+        
+          - name: Configuration AWS
+            run: bash infrastructure/scripts/aws_configuration.sh
+            env:
+                AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
+                AWS_PROFILE: ${{ secrets.AWS_PROFILE }}
+                AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+                AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+                AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+          
+          - name: Set up Terraform and Terragrunt
+            run: |
+                bash infrastructure/scripts/tf_tgswitch_install.sh
+                make tf 
+            
+          - name: Terragrunt initlization
+            run: make init
+          
+          - name: ECR Repo Code
+            working-directory: infrastructure/environments/dev/
+            run: |
+                export TF_VAR_aws_profile=satyam TF_VAR_aws_region=ap-south-1 TF_VAR_tf_bucket=satyam-terragrunts TF_VAR_prefix=project-dev
+                terragrunt apply -auto-approve -target=module.ecr_repo
+
+          - name: Image Push Script
+            run: |
```

Comment:
- you should create a make target for this command
- use `|` only when you need to run multiple commands in same step